src/hotspot/cpu/zero/c2_MacroAssembler_zero.hpp

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
This file is not needed. Please remove it.

Reply:
Done.